worker/config/config.go

```diff
@@ -46,6 +46,12 @@ type Config struct {
 	Handler_cache_size int `json:"handler_cache_size"` //kb
 	Import_cache_size  int `json:"import_cache_size"`  //kb
 
+	// average memory usage for each handler
+	Average_handler_memory_usage int `json:"average_handler_memory_usage"` //kb
```

Comment:
Let's get rid of this as an option.  The dynamic calculation you have seems better, and I don't think the ability to override that with a constant is worth the complexity.

Reply:
Updated in the new commit.